### ISSUE TITLE
Swallowing worsens tinnitus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10145]
**Preferred term label:**
Swallowing worsens tinnitus

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK519055/ "During swallowing, the TVP is activated to assist in elevating the palate. Due to this dual innervation by CNV, the TT muscle can be activated and possibly cause tinnitus in some patients."

This is a very specific term, but a more general term might be confused with https://hpo.jax.org/app/browse/term/HP:0032043

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000042
- label: Tinnitus exacerbated by swallowing
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: PMID:30085597
- def: Tinnitus (sound heard by a subjhect when no corresponding external sound is present) that is made worse by swallowing.
- comment: During swallowing, the tensor veli palatini (TVP) muscle is activated to assist in elevating the palate. Due to this dual innervation by CNV, the tensor tympani (TT) muscle can be activated and possibly cause tinnitus in some patients.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10145
- synonyms:
